tests/everflow/everflow_test_utilities.py

```diff
@@ -378,6 +378,12 @@ def setup_info(duthosts, rand_one_dut_hostname, tbinfo, request, topo_scenario):
     time.sleep(60)
 
 
+@pytest.fixture(scope="module")
+def dut_asic_type(duthosts, rand_one_dut_hostname, tbinfo, request, topo_scenario):
```

Comment:
Why this fixture depends on `tbinfo`, `request`, etc?

Reply:
Removed